src/components/relationships-panel/RelationshipsPanel.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <v-card flat id="relationships-panel" class="pb-4 pl-4">
+    <v-card flat rounded="sm" class="relationships-vcard px-4 mt-3 mr-9 ml-4">
+      <v-row no-gutters class="align-center mt-5">
+
+        <v-col cols="4">
+          <v-checkbox id="heir-legal-rep-checkbox" class="mt-0" v-model="selectedRelationships"
+            :value="RelationshipTypes.HEIR_LEGAL_REP" :label="RelationshipTypes.HEIR_LEGAL_REP"
+            :rules="relationshipRules" />
+        </v-col>
+
+        <v-col cols="4">
+          <v-checkbox id="officer-checkbox" class="mt-0" v-model="selectedRelationships"
+            :value="RelationshipTypes.OFFICER" :label="RelationshipTypes.OFFICER" :rules="relationshipRules" />
+        </v-col>
+
+        <v-col cols="4">
+          <v-checkbox id="director-checkbox" class="mt-0" v-model="selectedRelationships"
+            :value="RelationshipTypes.DIRECTOR" :label="RelationshipTypes.DIRECTOR" :rules="relationshipRules" />
+        </v-col>
+
+        <v-col cols="4">
+          <v-checkbox id="shareholder-checkbox" class="mt-0" v-model="selectedRelationships"
+            :value="RelationshipTypes.SHAREHOLDER" :label="RelationshipTypes.SHAREHOLDER" :rules="relationshipRules" />
+        </v-col>
+
+        <v-col cols="4">
+          <v-checkbox id="court-ordered-party-checkbox" class="mt-0" v-model="selectedRelationships"
+            :value="RelationshipTypes.COURT_ORDERED_PARTY" :label="RelationshipTypes.COURT_ORDERED_PARTY"
+            :rules="relationshipRules" />
+        </v-col>
+      </v-row>
+    </v-card>
+  </v-card>
+</template>
+<script lang="ts">
+
+import Vue from 'vue'
+import { Component, Watch, Emit, Prop } from 'vue-property-decorator'
+import { RelationshipTypes } from '@/enums/relationship-types'
```

Comment:
You need to import the enum like this (otherwise the project that imports this component will have to provide the enum in its own enum folder):
```
import { RelationshipTypes } from '@bcrs-shared-components/enums'
```

Reply:
Oh okay I didn't know that. Will do.

Reply:
We do have some components that weren't done this say, so they require an interface/enum/mixin/whatever both here for Storybook and also in the parent project :(  I guess we'll clean them up as we touch them.

Reply:
Er, I don't see any existing shared components that do what I just said we should do. Maybe it doesn't work...?

Reply:
I was checking all the other components that's been already done and I saw many of them depend on shared stuff like an enum or interface like you said!